src/main/plugins/org.dita.html5/xsl/topic.xsl

```diff
@@ -2150,12 +2161,14 @@ See the accompanying LICENSE file for applicable license.
   
   <!-- Catch footnotes that should appear at the end of the topic, and output them. -->
   <xsl:template match="*[contains(@class, ' topic/fn ')]" mode="genEndnote">
-    <div class="p">
```

Comment:
Don't know if removing this was intentional - I believe it will cause footnotes to be stacked slightly closer together by default, without the extra line break of paragraph-like content? CSS can still have control by matching `fn` just noted this seemed unrelated